client/src/app/project-create-edit/schemas/project-metadata-schema.json

```diff
@@ -396,12 +396,13 @@
       "description": "Publications resulting from this project.",
       "items": {
         "$schema": "http://json-schema.org/draft-07/schema#",
-        "$id": "https://schema.dev.data.humancellatlas.org/module/project/6.0.0/publication",
+        "$id": "https://schema.dev.data.humancellatlas.org/module/project/7.0.0/publication",
         "description": "Information about a journal article, book, web page, or other external available documentation for a project.",
         "additionalProperties": false,
         "required": [
           "authors",
-          "title"
+          "title",
+          "official_hca_publication"
```

Comment:
Should we keep this list sorted?

Reply:
Hmmm it's the same as https://schema.humancellatlas.org/module/project/7.0.0/publication

Reply:
OK. Let's keep it as is for now.
Keeping them sorted helps merging using diff tools.